x-pack/plugins/security_solution/public/resolver/data_access_layer/factory.ts

```diff
@@ -25,7 +25,7 @@ export function dataAccessLayerFactory(
      * Used to get non-process related events for a node.
      */
     async relatedEvents(entityID: string): Promise<ResolverRelatedEvents> {
-      return context.services.http.get(`/api/endpoint/resolver/${entityID}/events`, {
+      return context.services.http.post(`/api/endpoint/resolver/${entityID}/events`, {
```

Comment:
@bkimmel @kqualters-elastic @michaelolo24 any ideas if there are other places I need to change for related events `get` -> `post`?